### DISCUSSION TITLE
Input Modules

### DISCUSSION THREAD

**Discussion by TheTechromancer** [blacklanternsecurity/bbot#722] [Ideas]
Similar to how we have output modules for POSTing to an HTTP endpoint, etc., it would be nice to have input modules so that you can run BBOT (perhaps in agent mode?) and be able to send it targets on the fly.

![Image](https://user-images.githubusercontent.com/20261699/189703478-3f9fd82b-d1a7-4ebb-ae27-201ea6ad1d3f.png)

**Comment by aconite33** [comment]
Please review with @kerrymilan before proceeding.
